lib/sport_ngin_aws_auditor/ec2_instance.rb

```diff
@@ -52,14 +62,17 @@ def get_more_info(instances, client)
       private :get_more_info
     end
 
-    attr_accessor :id, :name, :platform, :availability_zone, :scope, :instance_type, :count, :stack_name, :tag_value, :tag_reason, :expiration_date, :count_remaining
-    def initialize(ec2_instance, tag_name, count=1)
+    attr_accessor :id, :name, :platform, :availability_zone, :scope, :instance_type, :count, :stack_name, :tag_value, :tag_reason, :expiration_date, :count_remaining, :ec2_classic_support
+    def initialize(ec2_instance, tag_name, ec2_classic_support, count=1)
```

Comment:
Is creating a new argument here necessary?  It looks like your initialize method could call the new ec2_classic_support method directly to get the value.